### PR TITLE
Add option 'initialIndex'

### PR DESCRIPTION
**Pull request by archive64**
This allows the user to specify an initial slide that should display when lory is first initialized.

Please ignore the files in `dist`, since they're generated with a build command.

### CODE REVIEW

File: src/lory.js [hunk 1]
```diff
@@ -219,9 +219,11 @@ export function lory (slider, opts) {
             classNamePrevCtrl,
             classNameNextCtrl,
             enableMouseEvents,
-            classNameActiveSlide
+            classNameActiveSlide,
+            initialIndex
         } = options;
 
+        index = initialIndex;
```
Comment (danielbeardsley) [inline src/lory.js:226]: Does this not create a global variable anymore?

**Comment by danielbeardsley** [comment]
CR 👍

**Comment by danielbeardsley** [comment]
Should we open a pull on the parent repo too?

File: src/lory.js (hunk 1 above)
Comment (archive64) [reply src/lory.js:226]: It's defined in the outer scope, on line 25.

**Comment by archive64** [comment]
I did! :) https://github.com/meandmax/lory/pull/630

**Comment by tvler** [comment]
CR 👍

**Pull request merged**